package.json

```diff
@@ -265,6 +265,20 @@
           "invoice.item.cancel"
         ]
       },
+      {
+        "permissionName": "ui-invoice.voucherExport",
```

Comment:
should settings permission be updated?

Reply:
IMO it's better to update permissions when all feature is moved to the new screen. Also voucher export list should be removed from settings.